### DISCUSSION TITLE
[lodash] predicates types don't support truthy/falsy return values

### DISCUSSION THREAD

**Discussion by robatwilliams** [DefinitelyTyped/DefinitelyTyped#56241] [Issues with a @types package]
The library supports returning truthy/falsy, but the typings sometimes (always?) want a Boolean.

e.g. `_.filter()`

I would recommend explicit true/false, but this inconsistency in the types is causing additional friction when converting existing JS code to TypeScript.

[a comment by an automated account is not shown]

**Comment by yesennes** [comment]
Looks like this is the same in root cause to https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/55403 which I made to reopen the old issue of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25758. Your title states it best though. It might be about time for a fork, the owners don't seem very responsive.
